services/QuillLMS/app/views/pages/evidence_tool.erb

```diff
@@ -8,123 +15,136 @@
+        </div>
       </div>
       <div class="topic-section">
-        <p class="topic">Science Topics</p>
-        <img alt="photograph of a cow" class="lazy" loading="lazy" data-src="https://assets.quill.org/images/evidence/home_page/EvidenceLandingPage_Image_Cow.webp" src="https://assets.quill.org/images/evidence/home_page/EvidenceLandingPage_Image_Cow.webp" />
-        <p class="activity-title">"How Does Eating Meat Impact Global Warming?"</p>
-        <a class="q-button text-quill-teal bg-white" href='/evidence/#/play?uid=176' rel="noopener noreferrer" target="_blank">View a sample activity</a>
+        <img alt="photograph of a cow" class="lazy" loading="lazy" data-src="https://assets.quill.org/images/evidence/home_page/evidence-tool-science-photo.png" src="https://assets.quill.org/images/evidence/home_page/evidence-tool-science-photo.png" />
+        <div class="topic-container">
+          <p class="topic">Interdisciplinary Science:</p>
+          <p class="subtopic">AI Knowledge</p>
+        </div>
+        <p class="activity-title">"How Are Marine Biologists Using Machine Learning to Protect Coral Reefs?"</p>
+        <div class="buttons-container">
+          <a class="quill-button contained medium teal focus-on-light" href='/assign/interdisciplinary-science/building-ai-knowledge' rel="noopener noreferrer" target="_blank">See all AI Knowledge activities</a>
+        </div>
       </div>
       <div class="topic-section">
-        <p class="topic">Social Studies Topics</p>
-        <img alt="photograph of the Statue of Liberty" class="lazy" loading="lazy" data-src="https://assets.quill.org/images/evidence/home_page/world_history.png" src="https://assets.quill.org/images/evidence/home_page/world_history_map.png" />
-        <p class="activity-title">World History</p>
-        <a class="q-button text-quill-teal bg-white" href='/social-studies' rel="noopener noreferrer" target="_blank">See all social studies activities</a>
+        <img alt="photograph of the Statue of Liberty" class="lazy" loading="lazy" data-src="https://assets.quill.org/images/evidence/home_page/evidence-tool-social-studies-photo.png" src="https://assets.quill.org/images/evidence/home_page/evidence-tool-social-studies-photo.png" />
+        <div class="topic-container">
+          <p class="topic">Social Studies:</p>
+          <p class="subtopic">World History</p>
+        </div>
+        <p class="activity-title">"How Did British Women Workers Fight for Their Rights in the 19th Century?"</p>
+        <div class="buttons-container">
+          <a class="quill-button sample-activity outlined medium teal focus-on-light" href='/evidence/#/play?anonymous=true&uid=457&session=d7983502-c657-449b-ba55-401cd2faa0c9' rel="noopener noreferrer" target="_blank">Try sample activity</a>
```

Comment:
Do we want a session id here? That seems strange.

Reply:
probably not... this was copied over from the old tool page, I'll remove it